ledger-service/cli-opts/src/main/scala/cliopts/Logging.scala

```diff
@@ -19,9 +54,26 @@ object Logging {
       .getOrElse(throw new java.lang.IllegalArgumentException(s"Unknown logging level $level"))
   }
 
+  @SuppressWarnings(Array("org.wartremover.warts.All"))
+  private implicit val scoptLogEncoder: scopt.Read[LogEncoder] =
+    scopt.Read.reads { encoder =>
+      Either
+        .cond(
+          KnownLogEncoders.contains(encoder.toLowerCase()),
```

Comment:
maybe just make the pattern match exhaustive and handle the case where it’s not one of the two strings there instead of having a separate check upfront?